lib/actions/adapterActions.js

```diff
@@ -1120,7 +1161,7 @@ export function openAdapter(port, versionInfo) {
                 }
 
                 logger.info(`Connectivity firmware version ${versionInfo.version} detected`);
-                logger.debug('Connectivity firmware info: ' +
+                logger.info('Connectivity firmware info: ' +
                     `sdBleApiVersion: ${versionInfo.sdBleApiVersion}, ` +
                     `baudRate: ${versionInfo.baudRate}, ` +
                     `transportType: ${versionInfo.transportType}`);
```

Comment:
Could merge these so that it logs firmware version, softdevice version, baud rate, and transport type in a single log message.